mavros/include/mavros/frame_tf.h

```diff
@@ -62,6 +62,8 @@ enum class StaticTF {
 	ENU_TO_NED,		//!< change from expressed WRT ENU frame to WRT NED frame
 	AIRCRAFT_TO_BASELINK,	//!< change from expressed WRT aircraft frame to WRT to baselink frame
 	BASELINK_TO_AIRCRAFT,	//!< change from expressed WRT baselnk to WRT aircraft
+	ABSOLUTE_FRAME_AIRCRAFT_TO_BASELINK,
```

Comment:
Can you please add thr description comment in these two, as in the others on top? Thanks!